packages/opencensus-instrumentation-http2/src/http2.ts

```diff
@@ -23,6 +23,7 @@ import {
   SpanKind,
   TraceOptions,
 } from '@opencensus/core';
+import 'http';
```

Comment:
Could you add a comment for why this line is needed to avoid the circular import? And similar for the other case below?